### ISSUE TITLE
POJAVLAUNCHER PLAY STORE VERSION CRASH

### ISSUE THREAD

**Issue by kaicryptx** [PojavLauncherTeam/PojavLauncher#3637]
### Describe the bug

When i try play any version the version loads but redirect in login screen

### The log file and images/videos

I dont have option to upload file

### Steps To Reproduce

```markdown
1. Open and go to play any version and then it automatically redirect into pojav login
```

### Expected Behavior

Plz hlp

### Platform

```markdown
- Device model: REALME C3 
- CPU architecture: aarch64
- Android version: 10
- PojavLauncher version: play store
```

**Comment by MazeXP** [comment]
Will also be related to #3634

**Comment by GaucovyAero** [comment]
just wait for it to get fixed i guess, or just download build #3997 in Actions for now.

**Comment by GaucovyAero** [comment]
fixed in the latest play store build
